lib/zoom/actions/meeting.rb

```diff
@@ -97,6 +97,16 @@ def livestream(*args)
         options.require(%i[meeting_id stream_url stream_key]).permit(%i[page_url])
         Utils.parse_response self.class.patch("/meetings/#{options[:meeting_id]}/livestream", body: options.except(:meeting_id), headers: request_headers)
       end
+
+      def meeting_invitation(*args)
+        options = Zoom::Params.new(Utils.extract_options!(args))
+        options.require(%i[meeting_id])
```

Comment:
Minor style adjustment (working on making this consistent across the codebase): Only use symbol array literals for arrays of 2 or more. Thanks!

```suggestion
        options.require(:meeting_id)
```